src/modules/rover_ackermann/module.yaml

```diff
@@ -175,3 +190,25 @@ parameters:
         increment: 0.01
         decimal: 2
         default: -1
+
+      RA_MAX_ACCEL:
+        description:
+          short: Maximum acceleration for the rover
+        type: float
+        unit: m/s^2
+        min: -1
+        max: 100
+        increment: 0.01
+        decimal: 2
+        default: -1
+
+      RA_MAX_STR_RATE:
+        description:
+          short: Maximum steering rate for the rover
+        type: float
+        unit: rad/s
```

Comment:
I think params should in general be deg and not rad, it's much simpler for most to get a feeling for it.

Reply:
Makes sense. On this topic I found something weird: The parameter `RA_MAX_STR_ANG` is also defined with the unit `rad` but in the qgc/amc UI it shows up as `degree`. This is actually a nice feature it is just odd that it doesn't work with `rad/s`.

Reply:
Implemented in [f8f7084](https://github.com/PX4/PX4-Autopilot/pull/23310/commits/f8f70842f2da6676f5ac360dd19987efad4c950b)